device/iot-device-samples/send-receive-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/SendReceive.java

```diff
@@ -239,15 +239,20 @@ else if (protocolStr.equals("amqps_ws"))
 
         System.out.println("Updated token expiry time to " + time);
 
+        String deviceId = "MyJavaDevice";
+        double temperature = 0.0;
```

Comment:
We use deviceId in all SDK samples for all other languages, e.g. https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/samples/iothub_client_sample_amqp/iothub_client_sample_amqp.c (for C), or https://github.com/Azure/azure-iot-sdk-node/blob/master/device/samples/simple_sample_device.js (for Node), etc. Also, this is the structure that we use at docs.microsoft.com for all the samples for the physical devices (e.g. https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-arduino-huzzah-esp8266-get-started)